### ISSUE TITLE
`test/proposal` has 0/? assertions passing. update them

### ISSUE THREAD

**Issue by Connoropolous** [holo-rea/holo-rea#251]
```
cd test
WASM_LOG=debug RUST_LOG=error RUST_BACKTRACE=1 npx tape proposal/**/*.js
```

**Comment by Connoropolous** [comment]
Assessment: hasn't been updated to new tryorama

Output:

```
18:29:44 [tryorama] info: Using the following settings from environment variables:
18:29:44 [tryorama] info: {
  "adminInterfaceId": "tryorama-interface-admin",
  "appInterfaceId": "tryorama-interface-app",
  "stateDumpOnError": true,
  "zomeCallTimeoutMs": 90000,
  "conductorTimeoutMs": 125000,
  "strictConductorTimeout": false,
  "chooseFreePort": false,
  "portRange": [
    33000,
    34000
  ],
  "legacy": false,
  "singletonAppId": "TRYORAMA_APP",
  "holochainPath": "holochain",
  "lairPath": "lair-keystore"
}
18:29:44 [tryorama] debug: About to execute 1 tests
18:29:44 [tryorama] debug: Executing test: Proposal record API
18:29:44 [tryorama] debug: About to execute 1 tests
18:29:44 [tryorama] debug: Executing test: ProposedIntent external link
18:29:44 [tryorama] debug: About to execute 1 tests
18:29:44 [tryorama] debug: Executing test: ProposedTo record API
TAP version 13
# Proposal record API
18:29:44 [tryorama] debug: api.players: creating players
18:29:44 [tryorama] error: Test error: 'Error: Config for player at index 0 contains something other than a function. Either use Config.gen to create a seed function, or supply one manually.\n' +
  '    at /Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/api.ts:56:15\n' +
  '    at Array.forEach (<anonymous>)\n' +
  '    at ScenarioApi.players (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/api.ts:54:19)\n' +
  '    at buildPlayer (/Users/connor/code/hrea/test/init.js:90:35)\n' +
  '    at /Users/connor/code/hrea/test/proposal/test_proposal_crud.js:31:23\n' +
  '    at p (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:126:35)\n' +
  '    at Test.<anonymous> (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:127:7)\n' +
  '    at Test.bound [as _cb] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)\n' +
  '    at Test.run (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:106:10)\n' +
  '    at Test.bound [as run] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)'
not ok 1 Test threw an exception. See output for details.
  ---
    operator: fail
    at: <anonymous> (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:137:13)
    stack: |-
      Error: Test threw an exception. See output for details.
          at Test.assert [as _assert] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:275:54)
          at Test.bound [as _assert] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)
          at Test.fail (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:368:10)
          at Test.bound [as fail] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)
          at /Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:137:13
          at processTicksAndRejections (node:internal/process/task_queues:96:5)
          at runNextTicks (node:internal/process/task_queues:65:3)
          at processImmediate (node:internal/timers:437:9)
  ...
18:29:44 [tryorama] debug: Test failed: Proposal record API Error: Config for player at index 0 contains something other than a function. Either use Config.gen to create a seed function, or supply one manually.
    at /Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/api.ts:56:15
    at Array.forEach (<anonymous>)
    at ScenarioApi.players (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/api.ts:54:19)
    at buildPlayer (/Users/connor/code/hrea/test/init.js:90:35)
    at /Users/connor/code/hrea/test/proposal/test_proposal_crud.js:31:23
    at p (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:126:35)
    at Test.<anonymous> (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:127:7)
    at Test.bound [as _cb] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)
    at Test.run (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:106:10)
    at Test.bound [as run] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32) {
  [stack]: 'Error: Config for player at index 0 contains something other than a function. Either use Config.gen to create a seed function, or supply one manually.\n' +
    '    at /Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/api.ts:56:15\n' +
    '    at Array.forEach (<anonymous>)\n' +
    '    at ScenarioApi.players (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/api.ts:54:19)\n' +
    '    at buildPlayer (/Users/connor/code/hrea/test/init.js:90:35)\n' +
    '    at /Users/connor/code/hrea/test/proposal/test_proposal_crud.js:31:23\n' +
    '    at p (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:126:35)\n' +
    '    at Test.<anonymous> (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:127:7)\n' +
    '    at Test.bound [as _cb] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)\n' +
    '    at Test.run (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:106:10)\n' +
    '    at Test.bound [as run] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)',
  [message]: 'Config for player at index 0 contains something other than a function. Either use Config.gen to create a seed function, or supply one manually.'
}
18:29:44 [tryorama] debug: Cleaning up test: Proposal record API
18:29:44 [tryorama] debug: Calling Api._cleanup. description: Proposal record API
18:29:44 [tryorama] debug: Finished with test: Proposal record API
# ProposedIntent external link
18:29:44 [tryorama] debug: api.players: creating players
18:29:44 [tryorama] error: Test error: 'Error: Config for player at index 0 contains something other than a function. Either use Config.gen to create a seed function, or supply one manually.\n' +
  '    at /Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/api.ts:56:15\n' +
  '    at Array.forEach (<anonymous>)\n' +
  '    at ScenarioApi.players (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/api.ts:54:19)\n' +
  '    at buildPlayer (/Users/connor/code/hrea/test/init.js:90:35)\n' +
  '    at /Users/connor/code/hrea/test/proposal/test_proposedintent_crud.js:32:23\n' +
  '    at p (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:126:35)\n' +
  '    at Test.<anonymous> (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:127:7)\n' +
  '    at Test.bound [as _cb] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)\n' +
  '    at Test.run (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:106:10)\n' +
  '    at Test.bound [as run] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)'
not ok 2 Test threw an exception. See output for details.
  ---
    operator: fail
    at: <anonymous> (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:137:13)
    stack: |-
      Error: Test threw an exception. See output for details.
          at Test.assert [as _assert] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:275:54)
          at Test.bound [as _assert] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)
          at Test.fail (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:368:10)
          at Test.bound [as fail] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)
          at /Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:137:13
          at processTicksAndRejections (node:internal/process/task_queues:96:5)
          at runNextTicks (node:internal/process/task_queues:65:3)
          at processImmediate (node:internal/timers:437:9)
  ...
18:29:44 [tryorama] debug: Test failed: ProposedIntent external link Error: Config for player at index 0 contains something other than a function. Either use Config.gen to create a seed function, or supply one manually.
    at /Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/api.ts:56:15
    at Array.forEach (<anonymous>)
    at ScenarioApi.players (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/api.ts:54:19)
    at buildPlayer (/Users/connor/code/hrea/test/init.js:90:35)
    at /Users/connor/code/hrea/test/proposal/test_proposedintent_crud.js:32:23
    at p (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:126:35)
    at Test.<anonymous> (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:127:7)
    at Test.bound [as _cb] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)
    at Test.run (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:106:10)
    at Test.bound [as run] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32) {
  [stack]: 'Error: Config for player at index 0 contains something other than a function. Either use Config.gen to create a seed function, or supply one manually.\n' +
    '    at /Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/api.ts:56:15\n' +
    '    at Array.forEach (<anonymous>)\n' +
    '    at ScenarioApi.players (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/api.ts:54:19)\n' +
    '    at buildPlayer (/Users/connor/code/hrea/test/init.js:90:35)\n' +
    '    at /Users/connor/code/hrea/test/proposal/test_proposedintent_crud.js:32:23\n' +
    '    at p (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:126:35)\n' +
    '    at Test.<anonymous> (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:127:7)\n' +
    '    at Test.bound [as _cb] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)\n' +
    '    at Test.run (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:106:10)\n' +
    '    at Test.bound [as run] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)',
  [message]: 'Config for player at index 0 contains something other than a function. Either use Config.gen to create a seed function, or supply one manually.'
}
18:29:44 [tryorama] debug: Cleaning up test: ProposedIntent external link
18:29:44 [tryorama] debug: Calling Api._cleanup. description: ProposedIntent external link
18:29:44 [tryorama] debug: Finished with test: ProposedIntent external link
# ProposedTo record API
18:29:44 [tryorama] debug: api.players: creating players
18:29:44 [tryorama] error: Test error: 'Error: Config for player at index 0 contains something other than a function. Either use Config.gen to create a seed function, or supply one manually.\n' +
  '    at /Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/api.ts:56:15\n' +
  '    at Array.forEach (<anonymous>)\n' +
  '    at ScenarioApi.players (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/api.ts:54:19)\n' +
  '    at buildPlayer (/Users/connor/code/hrea/test/init.js:90:35)\n' +
  '    at /Users/connor/code/hrea/test/proposal/test_proposedto_crud.js:25:23\n' +
  '    at p (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:126:35)\n' +
  '    at Test.<anonymous> (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:127:7)\n' +
  '    at Test.bound [as _cb] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)\n' +
  '    at Test.run (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:106:10)\n' +
  '    at Test.bound [as run] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)'
not ok 3 Test threw an exception. See output for details.
  ---
    operator: fail
    at: <anonymous> (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:137:13)
    stack: |-
      Error: Test threw an exception. See output for details.
          at Test.assert [as _assert] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:275:54)
          at Test.bound [as _assert] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)
          at Test.fail (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:368:10)
          at Test.bound [as fail] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)
          at /Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:137:13
          at processTicksAndRejections (node:internal/process/task_queues:96:5)
          at runNextTicks (node:internal/process/task_queues:65:3)
          at processImmediate (node:internal/timers:437:9)
  ...
18:29:44 [tryorama] debug: Test failed: ProposedTo record API Error: Config for player at index 0 contains something other than a function. Either use Config.gen to create a seed function, or supply one manually.
    at /Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/api.ts:56:15
    at Array.forEach (<anonymous>)
    at ScenarioApi.players (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/api.ts:54:19)
    at buildPlayer (/Users/connor/code/hrea/test/init.js:90:35)
    at /Users/connor/code/hrea/test/proposal/test_proposedto_crud.js:25:23
    at p (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:126:35)
    at Test.<anonymous> (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:127:7)
    at Test.bound [as _cb] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)
    at Test.run (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:106:10)
    at Test.bound [as run] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32) {
  [stack]: 'Error: Config for player at index 0 contains something other than a function. Either use Config.gen to create a seed function, or supply one manually.\n' +
    '    at /Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/api.ts:56:15\n' +
    '    at Array.forEach (<anonymous>)\n' +
    '    at ScenarioApi.players (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/api.ts:54:19)\n' +
    '    at buildPlayer (/Users/connor/code/hrea/test/init.js:90:35)\n' +
    '    at /Users/connor/code/hrea/test/proposal/test_proposedto_crud.js:25:23\n' +
    '    at p (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:126:35)\n' +
    '    at Test.<anonymous> (/Users/connor/code/hrea/node_modules/.pnpm/@holochain+tryorama@0.4.10/node_modules/@holochain/tryorama/src/middleware.ts:127:7)\n' +
    '    at Test.bound [as _cb] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)\n' +
    '    at Test.run (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:106:10)\n' +
    '    at Test.bound [as run] (/Users/connor/code/hrea/node_modules/.pnpm/tape@4.15.0/node_modules/tape/lib/test.js:89:32)',
  [message]: 'Config for player at index 0 contains something other than a function. Either use Config.gen to create a seed function, or supply one manually.'
}
18:29:44 [tryorama] debug: Cleaning up test: ProposedTo record API
18:29:44 [tryorama] debug: Calling Api._cleanup. description: ProposedTo record API
18:29:44 [tryorama] debug: Finished with test: ProposedTo record API

1..3
# tests 3
# pass  0
# fail  3
```

**Comment by pospi** [comment]
Fixed in 3574f1c2..449f2376 (inclusive)
